Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/resources/exportApi.js

```diff
@@ -0,0 +1,10 @@
+angular.module('virtoCommerce.exportModule')
+    .factory('virtoCommerce.exportModule.exportModuleApi', ['$resource', function ($resource) {
+        return $resource('', null, {
+            getProviders: { method: 'GET', isArray: true, url: 'api/export/providers'},
+            getKnowntypes: { method: 'GET', isArray: true ,url: 'api/export/knowntypes' },
+            runExport: {method: 'POST', url: 'api/export/run'},
+            download: { method: 'GET', url: '"download/{fileName}' },
```

Comment:
Not sure if we need this, as download is performed via link.
How is it implemented in similar places in old export?